src/redis/object.h

```diff
@@ -8,13 +8,14 @@
 #include "quicklist.h"
 
 /* The actual Redis Object */
-#define OBJ_STRING 0U    /* String object. */
-#define OBJ_LIST 1U      /* List object. */
-#define OBJ_SET 2U       /* Set object. */
-#define OBJ_ZSET 3U      /* Sorted set object. */
-#define OBJ_HASH 4U      /* Hash object. */
-#define OBJ_MODULE 5U    /* Module object. */
-#define OBJ_STREAM 6U    /* Stream object. */
+#define OBJ_STRING 0U       /* String object. */
+#define OBJ_LIST 1U         /* List object. */
+#define OBJ_SET 2U          /* Set object. */
+#define OBJ_ZSET 3U         /* Sorted set object. */
+#define OBJ_HASH 4U         /* Hash object. */
+#define OBJ_MODULE 5U       /* Module object. */
+#define OBJ_STREAM 6U       /* Stream object. */
+#define OBJ_TYPES_COUNT 7U  /* How many types of objects there are. */
```

Comment:
I know it's messy but we also have `OBJ_JSON = 15` in the codebase.
I prefer avoid touching object.h with our changes.

Instead, we have "redis_aux.h" that we own and it can bridge the gap.
I suggest moving there `OBJ_JSON` and `OBJ_TYPES_COUNT` and revert this file.